text_generation/causal_lm/cpp/CMakeLists.txt

```diff
@@ -28,3 +28,11 @@ find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 target_link_libraries(speculative_decoding_lm PRIVATE openvino::runtime)
 set_target_properties(speculative_decoding_lm PROPERTIES CXX_STANDARD 17)
 set_target_properties(speculative_decoding_lm PROPERTIES CXX_STANDARD_REQUIRED ON)
+
+add_executable(chat_model_lm chat_model_lm.cpp)
+target_compile_definitions(chat_model_lm PRIVATE OPENVINO_TOKENIZERS_PATH=\"$<TARGET_FILE:openvino_tokenizers>\")
+target_include_directories(chat_model_lm PRIVATE ./)
+find_package(OpenVINO REQUIRED COMPONENTS Runtime)
+target_link_libraries(chat_model_lm PRIVATE openvino::runtime)
+set_target_properties(chat_model_lm PROPERTIES CXX_STANDARD 17)
+set_target_properties(chat_model_lm PROPERTIES CXX_STANDARD_REQUIRED ON)
```

Comment:
could you please also add GHA CI pipeline for this sample?

Reply:
maybe we can also move this sample to a dedicated folder `llm-chatbot` with its own README.md?

Reply:
Can we add id in the next PR?

Reply:
CI should be done in current PR